employees/templates/employees/project_report_detail.html

```diff
@@ -26,5 +26,7 @@ <h1>{{ UI_text.PAGE_TITLE.value }}{{ report.project }} ({{ report.date }})</h1>
 {% endblock %}
 
 {% block extra_script %}
-{{ form.media }}
+  {{ form.media }}
+  <script src="//code.jquery.com/jquery-3.3.1.js"></script>
```

Comment:
This one is already in `base.html` template, so it should be loaded in every template that extends `base.html` template. Are you sure it is needed too? If yes, it means something is fundamentally wrong with our scripts loading in templates.